benchmarks/upload_scribe.py

```diff
@@ -30,6 +30,7 @@ def format_message(self, field_dict):
             elif field in self.schema['float']:
                 message['float'][field] = float(value)
             else:
+
```

Comment:
This is unnecessary, isn't it?
```suggestion
```

Reply:
yes, this was a partial revert, code refactoring will be done in future fix in #61809